### ISSUE TITLE
We have disabled a test until Capybara updates

### ISSUE THREAD

**Issue by AshGDS** [alphagov/govuk_publishing_components#3777]
We have disabled a test which was causing every run of CI to fail. The test was checking that Accessibility issues on the component guide appear in the JS console. See https://github.com/alphagov/govuk_publishing_components/pull/3774

The failing test seems to be due to a change in `chromedriver` which I believe Capybara does not support yet. 

Therefore once Capybara has released a version higher than `3.39.2`, we should reenable the test again.

Here is the Capybara change that's waiting for a release: https://github.com/teamcapybara/capybara/pull/2726
